improver_tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -401,16 +350,8 @@ def test_radii_varying_with_lead_time_with_interpolation_check_data(self):
         which are required but were not specified within the 'radii'
         argument."""
 
-        expected = np.ones_like(self.multi_time_cube.data)
-        expected[0, 6:9, 6:9] = (
-            [0.91666667, 0.875, 0.91666667],
-            [0.875, 0.83333333, 0.875],
-            [0.91666667, 0.875, 0.91666667],
-        )
-
-        expected[1, 5:10, 5:10] = SINGLE_POINT_RANGE_3_CENTROID
-
-        expected[2, 4:11, 4:11] = (
+        expected = np.ones_like(self.cube.data)
```

Comment:
I've changed this test to only contain one leadtime rather than 3, but otherwise kept the test the same